packages/hash/frontend/src/components/layout/PageHeader/AccountDropdown.tsx

```diff
@@ -1,51 +1,141 @@
-          Sign Out
-        </button>
-      </Menu.Item>
-    </Menu.Items>
-  </Menu>
-);
+  user,
+}) => {
+  const theme = useTheme();
+
+  const buttonRef = useRef(null);
+
+  const [open, setOpen] = useState(false);
+
+  const id = open ? "account-popover" : undefined;
+
+  return (
+    <Box>
+      <Button
+        variant="transparent"
+        onClick={() => setOpen(!open)}
+        title={user.properties.shortname!}
+        className="flex items-center relative m-auto focus:outline-none"
+        ref={buttonRef}
+        sx={{
+          borderRadius: "100%",
+          boxShadow: open
+            ? "0px 0px 0px 2px #FFFFFF, 0px 0px 0px 5px #C1CFDE"
+            : "unset",
+          ":hover": {
+            boxShadow: "0px 0px 0px 2px #FFFFFF, 0px 0px 0px 5px #C1CFDE",
+          },
+        }}
+      >
+        {avatar ? (
+          <Box
+            component="img"
+            alt="avatar"
+            src={avatar}
+            sx={{ height: "32px", width: "32px", borderRadius: "100%" }}
+            className={tw`border border(solid gray-200)`}
+          />
+        ) : (
+          <Box
+            sx={{
+              height: "32px",
+              width: "32px",
+              borderRadius: "100%",
+              color: theme.palette.common.white,
+              background: theme.palette.blue[70],
+              display: "flex",
+              alignItems: "center",
+              justifyContent: "center",
+            }}
+          >
+            {user.properties.preferredName![0]}
+          </Box>
+        )}
+      </Button>
+      <Popover
+        id={id}
+        open={open}
+        anchorEl={buttonRef.current}
+        onClose={() => setOpen(false)}
+        anchorOrigin={{
+          vertical: "bottom",
+          horizontal: "right",
+        }}
+        transformOrigin={{
+          vertical: "top",
+          horizontal: "right",
+        }}
+        PaperProps={{
+          elevation: 4,
+          sx: {
+            width: 225,
+            borderRadius: "6px",
+            marginTop: 1,
+            border: `1px solid ${theme.palette.gray["20"]}`,
+          },
+        }}
+      >
+        <Box px={2} pt={1} pb={1.5}>
+          <Typography variant="smallCopy">
+            <strong>{user.properties.preferredName}</strong>
+          </Typography>
+          <Typography component="p" variant="microCopy">
+            @{user.properties.shortname!}
+          </Typography>
+        </Box>
+        <Divider />
+        <Box>
+          <Link noLinkStyle href="#" onClick={() => setOpen(false)}>
+            <ListItemButton
+              sx={{
+                padding: theme.spacing(1, 2),
+                m: 0.5,
+                borderRadius: 1,
+              }}
+            >
+              <Typography variant="smallCopy" sx={{ lineHeight: 1 }}>
+                Account Settings
+              </Typography>
+            </ListItemButton>
+          </Link>
+          <Divider />
+          <ListItemButton
```

Comment:
We should disable touch ripple for this component by default, you can see [how we do this in the BP repo](https://github.com/blockprotocol/blockprotocol/blob/main/site/src/theme/components/dataDisplay/MuiListItemButtonThemeOptions.ts)

Reply:
I think this should be done as part of this PR (or in a close follow-up) as from memory both @nonparibus and @MaggieAppleton dislike the MUI ripple effect:

https://user-images.githubusercontent.com/42802102/156571714-9ca6efd6-4ada-4adf-938e-5b05ab5298e6.mov

Reply:
I feel like a subtle version of it at greatly reduced opacity (maybe 30%?) would be fine - but as you say the full effect is a bit overbearing.

Reply:
Disabling this for now. Have added a task for future implementation and brainstorming on where this ripple can be used: https://app.asana.com/0/1201837296904616/1201910890445806/f